CoronaTracker.Core/Services/Processings/ExternalCountries/ExternalCountryProcessingService.Exceptions.cs

```diff
@@ -0,0 +1,66 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using CoronaTracker.Core.Models.ExternalCountries;
+using CoronaTracker.Core.Models.ExternalCountries.Exceptions;
+using CoronaTracker.Core.Models.Processings.ExternalCountries;
+using CoronaTracker.Core.Models.Processings.ExternalCountries.Exceptions;
+using Xeptions;
+
+namespace CoronaTracker.Core.Services.Processings.ExternalCountries
+{
+    public partial class ExternalCountryProcessingService
+    {
+        private delegate ValueTask<List<ExternalCountry>> ReturningExternalCountriesFunction();
+
+        private async ValueTask<List<ExternalCountry>>
+        TryCatch(ReturningExternalCountriesFunction returningExternalCountriesFunction)
```

Comment:
This needs a tab and you can also break from the input parameter instead of breaking after the return type.